lib/mix/tasks/firmware.burn.ex

```diff
@@ -99,7 +110,20 @@ defmodule Mix.Tasks.Firmware.Burn do
   end
 
   defp get_devs do
-    {result, 0} = System.cmd("fwup", ["--detect"])
+    {result, 0} = 
+      if is_wsl?() do
+        path = Path.expand("./") <> "/fwup_devices.txt"
+        path = conv_path_to_windows(path)
+        System.cmd("powershell.exe", ["-Command", "Start-Process powershell.exe -Verb runAs -Wait -ArgumentList \"fwup.exe -D | set-content -encoding UTF8 #{path}\""])
```

Comment:
I'm getting
```
Set-Content : A parameter cannot be found that matches parameter name 'encoding'.
At line:1 char:27
+ fwup.exe -D | set-content -encoding UTF8 /:tmecklem/src/windows/fwup_ ...
+                           ~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Set-Content], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.SetContentCommand
```

Reply:
It seems to have happened since my nerves app directory was inside the wsl user home directory and not somewhere else. I moved it to a location under /mnt/c and it worked.

Reply:
Hey @tmecklem, thanks for the review!
I pushed a commit that should have fixed this. Please let me know if it still doesn't work.